Cargo.audit.ignore

```diff
@@ -0,0 +1 @@
+RUSTSEC-2024-0370
```

Comment:
what dependency introduced that CVE?

Reply:
See https://github.com/omnect/omnect-cli/pull/134/commits/f0dbf9f641b1c1dc8185a752e4eab8805dfd0e32 for the details. It is a dependency of validator/validator_derive